frontend/VolunteerOne/screens/Profile/index.js

```diff
@@ -91,13 +101,26 @@ const ProfileScreen = ({ route, navigation }) => {
                     >
                       {isVolunteer ? "CONNECT" : "JOIN"}
```

Comment:
Actually, only show button with text "CONNECT" if the user is a Volunteer, otherwise, show only the follow/unfollow toggle btn. So volunteers should have 2 btns, and organizations should have 1 btn

Reply:
Done!